packages/server-ct/src/project-ct.ts

```diff
@@ -64,85 +60,17 @@ export class ProjectCt extends ProjectBase<ServerCt> {
     })
   }
 
+  _onError<Options extends Record<string, any>> (err: Error, options: Options) {
+    debug('plugins failed with error', err)
+
+    options.onError(err)
+  }
+
   _initPlugins (cfg, options) {
     // only init plugins with the
     // allowed config values to
     // prevent tampering with the
     // internals and breaking cypress
-    const allowedCfg = config.allowed(cfg)
```

Comment:
All of this was duplicated, now it lives in `project-base.ts` so both CT and E2E can use it. We need a few CT specific checks, since we use a dev server, but I think it's a lot better than having two separate places to do the same thing.